test/vm_tests/transformer/impl_generation_test.dart

```diff
@@ -328,6 +328,80 @@ main() {
           expect(transformedSource, contains(transformedLine));
         });
 
+        test('with builder compatible private props class', () {
+          final originalPrivateFooPropsLine = 'class _\$FooProps extends UiProps {}';
+          final originalPublicFooPropsLine = 'class FooProps extends UiProps with _\$FooPropsAccessorsMixin implements _\$FooProps';
```

Comment:
We are no longer extending `UiProps`/`UiState` and implementing the private class, instead we're just extending the private class:

```suggestion
          final originalPublicFooPropsLine = 'class FooProps extends _\$FooProps with _\$FooPropsAccessorsMixin {';
```

Reply:
Same changes for the State test below